modules/data/about_newtab.components.json

```diff
@@ -25,20 +23,40 @@
         "groups": []
     },
 
-    "top-sites-list": {
-        "selectorData": "top-sites-list",
+    "sponsored-site-card": {
+        "selectorData": "top-site-outer",
         "strategy": "class",
         "groups": [
-            "requiredForPage"
```

Comment:
Why was `requiredForPage` removed here? Just wondering

Reply:
Because this was useful for the Pocket tests, which are apparently no longer our team's jurisdiction, but when switching the Tile tests to use this POM, it caused them to timeout (because we weren't actually using the prefs that the full Pocket tests need)